app/kmqs/main.js

```diff
@@ -16,8 +16,8 @@ function load(u) {
 }
 
 if (config.kissmetricsKey) {
-  load('//i.kissmetrics.com/i.js');
-  load('//doug1izaerwt3.cloudfront.net/' + config.kissmetricsKey + '.1.js')
+  load('https://i.kissmetrics.com/i.js');
```

Comment:
well, i suspect it could detect if it is `file://` like cordova and then force https automatically.